testing/lib/workarea/storefront/integration_test.rb

```diff
@@ -66,7 +66,7 @@ def complete_checkout(email = nil, password = nil)
             credit_card: {
               number: '1',
               month:  1,
-              year:   2020,
+              year:   next_year,
```

Comment:
Why not `twenty_twenty`? :trollface:

Reply:
`alias_method twenty_twenty next_year`

accepting PRs...